ftplugin/k8s_top_nodes.lua

```diff
@@ -21,7 +22,7 @@ local function set_keymaps(bufnr)
     desc = "Top nodes",
     callback = function()
       nodes_top_view.View()
-      top_def.res_type = "nodes"
+      nodes_top_def.res_type = "nodes"
```

Comment:
https://github.com/Ramilito/kubectl.nvim/pull/426

Reply:
Reverted this!